src/styles/annotator/bucket-bar.scss

```diff
@@ -86,10 +88,17 @@
         border-right: solid transparent;
         margin-top: 0;
       }
+      & .label {
+        // Vertical alignment tweak to better center the label in the indicator
+        margin-top: -1px;
+      }
     }
 
     &.upper {
       border-radius: 2px 2px 4px 4px;
+      // Vertically center the element (which is 22px high) in the
```

Comment:
The "22px" value comes from the height of the `.upper` element plus the triangle created by the `::before` and `::after` pseudo-selectors?

Reply:
Correct. The entire element takes up 22px of space, vertically.